docs/notebooks/04_example_dissipative_qubit_reset.ipynb

```diff
@@ -756,25 +759,11 @@
    },
    "outputs": [],
    "source": [
-    "def TLS_onb_trg():\n",
-    "    \"\"\"Returns the tensor product of qubit target state\n",
-    "    and a basis for the TLS Hilbert space\"\"\"\n",
-    "    rho1 = qutip.Qobj(np.kron(rho_q_trg, np.diag([1, 0])))\n",
-    "    rho2 = qutip.Qobj(np.kron(rho_q_trg, np.diag([0, 1])))\n",
-    "    return [rho1, rho2]\n",
-    "\n",
-    "\n",
-    "TLS_onb = TLS_onb_trg()\n",
-    "\n",
-    "\n",
     "def chis_qubit(fw_states_T, objectives, tau_vals):\n",
     "    \"\"\"Calculate chis for the chosen functional\"\"\"\n",
     "    chis = []\n",
     "    for state_i_T in fw_states_T:\n",
-    "        chis_i = np.zeros(shape=(4, 4), dtype=np.complex_)\n",
-    "        for state_k in TLS_onb:\n",
-    "            a_i_k = krotov.optimize._overlap(state_i_T, state_k)\n",
-    "            chis_i += a_i_k * state_k\n",
+    "        chis_i = qutip.Qobj(np.kron(rho_q_trg, np.diag([1, 1])))\n",
     "        chis.append(qutip.Qobj(chis_i))\n",
```

Comment:
You're creating the `Qobj` twice here. Either define `chis_i = np.kron(...)` and then convert to `Qobj` in the next line or construct `chis_i` (which should probably be named `chi_i`) as a `Qobj` and then add it directly to `chis`. Or, do it in a single line.

Reply:
Commit with `--fixup` and then do an interactive rebase to keep the commits clean.

Reply:
Looks good! Can you do the interactive rebase to combine the fixup commit with the original "Fix wrong co-states definition in one example", and force-push?